dp-terraform/helm/rhacs-terraform/charts/observability/templates/01-operator-03-secret-alertmanager.yaml

```
@@ -10,12 +10,20 @@ stringData:
     route:
       receiver: managed-rhacs-pagerduty
       repeat_interval: 12h
+      group_by:
+      - cluster_id
       routes:
       - receiver: managed-rhacs-deadmanssnitch
```

This was not changed in this PR, but is this a typo?
```suggestion
      - receiver: managed-rhacs-deadmansswitch
```

---

I don't think this is affecting anything.